### PR TITLE
course: task 3 webpack

### PR DESCRIPTION
**Pull request by BorislavAnchev**
The WebPack task.

### CODE REVIEW

**Comment by BorislavAnchev** [comment]
Code ready for review.

**Comment by mgenov** [comment]
build folders (bin in this case) must be added to .gitignore.

**Comment by BorislavAnchev** [comment]
Fixes are ready.

**Comment by mgenov** [comment]
build folder is still not added to .gitignore.

**Comment by mgenov** [comment]
If I check out the project I'm able to build output files on my own:

image


so these artefacts should not be added in the version control system.

**Comment by BorislavAnchev** [comment]
Both 'bin' and 'build' folders should be ignored now. Initially I understood that only the 'bin' folder should be ignored.

**Comment by BorislavAnchev** [comment]
@mgenov, are there any other comments on this task?

**Comment by mgenov** [comment]
This branch cannot be rebased due to conflicts. 

image

**Comment by BorislavAnchev** [comment]
Conflicts should be fixed now.

File: 3WebPack/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "sass-loader": "^7.1.0",
+    "style-loader": "^0.23.1",
+    "webpack": "^4.29.6",
+    "webpack-cli": "^3.2.3",
+    "webpack-dev-server": "^3.7.2",
+    "webpack-merge": "^4.2.1",
+    "webpack-obfuscator": "^0.18.3"
+  },
+  "dependencies": {
+    "clean-webpack-plugin": "^3.0.0",
```
Comment (mgenov) [inline 3WebPack/package.json]: part of dev-dependencies.

**Pull request merged**